src/common/snippets/src/pass/transpose_decomposition.cpp

```diff
@@ -5,26 +5,32 @@
 #include <snippets/pass/transpose_decomposition.hpp>
 #include <snippets/itt.hpp>
 #include <snippets/snippets_isa.hpp>
-#include <snippets/tensor_descriptor.hpp>
+#include <snippets/port_descriptor.hpp>
 #include <ngraph/partial_shape.hpp>
 #include <ngraph/pattern/op/wrap_type.hpp>
 #include <ngraph/pass/manager.hpp>
-const std::set<std::vector<int>> ngraph::snippets::pass::TransposeDecomposition::supported_cases = {{0, 2, 3, 1}};
-ngraph::snippets::pass::TransposeDecomposition::TransposeDecomposition() {
+
+namespace ngraph {
+namespace snippets {
+namespace pass {
+
+const std::set<std::vector<int>> TransposeDecomposition::supported_cases = {{0, 2, 3, 1}};
+
+TransposeDecomposition::TransposeDecomposition() {
     MATCHER_SCOPE(TransposeDecomposition);
-    // todo: we need a special transformation that detects and propagates data access pattern to Parameters and Results
-    //  this is needed to communicate access pattern to the plugin node and op::Kernel
-    // This is the reason we match only to Parameter, this limitation could be relaxed if we propagate access pattern
-    // to the appropriate parameter
+    // Todo: we need a special transformation that detects and propagates data access pattern to Parameters and Results
+    //       this is needed to communicate access pattern to the plugin node and op::Kernel
+    //       This is the reason we match only to Parameter, this limitation could be relaxed if we propagate access pattern
+    //       to the appropriate parameter
```

Comment:
It looks like this Todo is obsolete, because we already have a layout propagation pass. Remove?

Reply:
We support propagation only if `Transpose` and `Parameter` have direct connection without operations between them.
I think the comment is more about that. Need to leave